src/js/storage/storage.js

```diff
@@ -21,10 +21,9 @@ export class Store {
 
   set state(state) {
     if (state) {
-      //Do not stringify value if value is already is a string ; to avoid adding "" around the string value
-      this.storage.setItem(this.key, typeof state === 'string' ? state : JSON.stringify(state));
```

Comment:
Revert of https://github.com/NoroffFEU/agency.noroff.dev/commit/86b2d794ee8fb4e4ad514c22f036abe383354ccd#diff-551b6093ecd9156be9fbb034108f4b1c4c0dda79477e186581e7b394abd6fddaR25 due to https://github.com/NoroffFEU/agency.noroff.dev/pull/544#discussion_r1446448390